src_assets/common/assets/web/styles/modal-glass.less

```
@@ -334,102 +243,71 @@
     border: 1px solid var(--modal-border-color);
-    border-radius: 12px;
+    border-radius: var(--ui-radius-md);
     overflow: hidden;
     margin-bottom: 0.75rem;
 
-    [data-bs-theme='light'] & {
-      background: #f5f8ff;
-      border-color: rgba(99, 102, 241, 0.2);
-      box-shadow: 0 3px 8px rgba(99, 102, 241, 0.08);
-    }
+    box-shadow: var(--ui-shadow-sm);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)